peepingtom/datablocks/multiblocks/sphereblock.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+from ..abstractblocks import SpatialBlock, MultiBlock
+from ..simpleblocks import PointBlock, PropertyBlock
+
+
+class SphereBlock(SpatialBlock, MultiBlock):
+    """
+    Represents a spheres defined by center and a radii
```

Comment:
this block represents multiple spheres, right? 'a spheres' should be changed to 'a set of spheres defined by centers and radii'

Reply:
Yup, done!